gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/study_point/StudyPointService.java

```diff
@@ -133,6 +141,10 @@ private void resetInitialPminPmax(Network network, ZonalData<Scalable> scalableZ
         LOGGER.info("Pmax and Pmin are reset to initial values for network {}", network.getNameOrId());
     }
 
+    private RaoRequest buildRaoRequest(String requestId, String cbcora, String cgm, String glsk) {
```

Comment:
It's better to use RaoRequest from com.farao_community.farao.rao_runner.api.resource
You cal also add the result destination directory in the RaoRequest